easybuild/tools/config.py

```diff
@@ -865,6 +867,46 @@ def log_path(ec=None):
     return log_file_format(return_directory=True, ec=ec, date=date, timestamp=timestamp)
 
 
+def get_failed_installs_build_dirs_path(ec):
+    """
+    Return the 'failed_installs_build_dirs_path',
+    the location where build directories are copied if installation failed
```

Comment:
I wouldn't repeat the function name here and only a single dir is returned for the `ec`:
```suggestion
    Return the location where the build directory is copied to if installation failed
```